src/icons/EyeClosedIcon.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+
+import { Icon } from '../components/Icon'
+
+const EyeClosedIcon = (props) => (
+  <Icon { ...props }>
+    <g transform="translate(1 8)">
+      <path d="M14.806 16.59c8.686 0 14.133-6.603 14.133-6.603.37-.403.375-1.06-.014-1.467 0 0-5.434-6.54-14.12-6.54C6.12 1.98.674 8.532.674 8.532c-.372.4-.376 1.06 0 1.454 0 0 5.447 6.602 14.132 6.602zm.194-.026c4.01 0 7.258-3.272 7.258-7.308S19.008 1.95 15 1.95c-4.01 0-7.258 3.27-7.258 7.306s3.25 7.308 7.258 7.308zm-9.92 2.192L24.92.244" stroke="#555" stroke-width="1.3"/><ellipse fill="#555" cx="14.758" cy="9.5" rx="3.629" ry="3.654"/>
```

Comment:
there is a defined fill for this - I think you can use `currentColor` (We are doing this for the stroke in the app and i think some of the other ones that we use a colored fill (the [warning](https://github.com/resmio/mantecao/blob/master/src/icons/WarningIcon.js#L8) icon I think https://css-tricks.com/cascading-svg-fill-color/